clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/PingMessage.java

```diff
@@ -3,12 +3,18 @@
 
 package com.microsoft.aspnet.signalr;
 
-class PingMessage extends HubMessage {
+class PingMessage extends HubMessage
+{
+    private static PingMessage instance = new PingMessage();
 
-    int type = HubMessageType.PING.value;
+    private PingMessage()
+    {
+    }
+
+    public static PingMessage getInstance() {return instance;}
```

Comment:
nit: formatting

Reply:
Do we always want a new line after a bracket? I'll add a spotless rule

Reply:
I think it's more about the spaces here

Reply:
Yeah I get that, but it's easier to normalize and write rules for if we just put newlines